src/components/shared/DonationSummary.vue

```diff
@@ -58,7 +58,7 @@ export default Vue.extend( {
 			return countryObject ? countryObject.countryFullName : '';
 		},
 		salutation: function () {
-			if ( !this.address.salutation ) {
+			if ( !this.address.salutation || this.address.salutation === 'No Salutation' || this.address.salutation === 'Keine Anrede' ) {
```

Comment:
I wonder if this can be checked against the salutation options that are used in the select options component? That way it won't have to be manually updated when/if we ever add a new language.

Reply:
So I had a look and you can remove this extra evaluation altogether and instead of returning `this.address.salutation` you can do something like this:

```js
salutation: function () {
	if ( !this.address.salutation ) {
		return '';
	}

	return this.$props.salutations.find( salutation => salutation.value === this.address.salutation )?.display;
},
```

Have a look in https://github.com/wmde/fundraising-app-frontend/blob/main/src/components/pages/donation_confirmation/DonationSummary.vue for an existing example.